components/Avatar/avatar.css

```
@@ -0,0 +1,25 @@
+.avatar {
+    border-radius: 50%;
+    width: 2.5rem;
+    height: 2.5rem;
+    margin: 0 10px;
+}
+
+.sm {
```

```suggestion
.avatar .sm {
```
You can name these like this, because a single '.sm' does not tell much information when reading the styles

---

ok